tests/Cache/FileHandlerTest.php

```diff
@@ -138,7 +138,10 @@ public function testWriteWritesCache()
         $handler->write($cache);
 
         $this->assertFileExists($file);
-        $this->assertSame($cache->toJson(), file_get_contents($file));
```

Comment:
some of the SCA tools asked to use file-dedicated assertion, but that assertion take content of that file as expected value, while for us it;s actual value we compare to expected json value.

for not having complain in the end